protein_ligand/screening_format.txt

```diff
@@ -0,0 +1,86 @@
+# The file name must begin with the letters "PHIP2" and then be followed by an underscore or dash.
+#
+# FILE FORMAT: All comment lines in this file (which begin with #) will be ignored.
+# Please use only UTF-8 characters in the non-comment fields. If your information (e.g. your name, etc.)
+# contains a non-UTF-8 character, you may note it in comments near that entry.
+#
+#
+# PREDICTIONS
+# The data in each prediction line should be structured as follows:
+# Fragment identifier, binding in S1 (true/false), binds in S2 (true/false), binds in S3 (true/false), binds in S4 (true/false), binds overall (true/false)
+# Any compounds not listed are assumed to be predicted as nonbinders.
+#
+#
+# The list of predictions must begin with the "Prediction:" keyword, as illustrated here.
+Predictions:
+F1, true, true, true, true, true
+F10, false, true, false, false, true
+F20, false, false, true, false, true
+F30, false, false, false, true, true
+F40, true, false, false, false, true
+F50, false, false, false, false, false
+
+#
+#
+# Please list your name, using only UTF-8 characters as described above. The "Participant name:" entry is required.
+Participant name:
+David Mobley
+#
+#
+# Please list your organization/affiliation, using only UTF-8 characters as described above.
+Participant organization:
+University of California, Irvine
+#
+#
+# Please provide a brief (40 character limit) informal yet informative name of the method used.
+# Following is sample text; please edit to your taste.
+# The "Name:" keyword is required, as shown here.
+# 40 character limit.
+Name:
+OEDocking-template
+#
+# All major software packages used and their versions
+# Following is sample text; please edit to your taste.
+# The "Software:" keyword is required.
+Software:
+OpenEye docking toolkit
+#
+# Methodology and computational details.
+# Level of detail should be at least that used in a publication.
+# Please include the values of key parameters, with units, and explain how any
+# statistical uncertainties were estimated.
+# Use as many lines of text as you need.
+# Please explicitly describe how you handle ions (e.g. counterions) and pKa effects
+# Following is sample text; please edit to your taste.
+# All text following the "Method:" keyword will be regarded as part of your free text methods description.
+Method:
+Compound conformers were generated with OpenEye Omega (version XYZ) with the OpenEye toolkits,
+then docked to the prepared receptor with the OpenEye docking toolkits using the
+python code snippet provided below, with the Chemgauss4 scoring function, with each site treated in a separate pass.
+(Insert code snippet here).
+Binders were predicted based on compounds scoring more favorably than a threshold score of (insert value)
+in any site.
+
+#
+#
+# METHOD CATEGORY SECTION
+#
+# State which method category your prediction method is better described as:
+# `Docking`, `Ligand-based`, `Simulation`, `ML`, or `Other`.

```

Comment:
Would you consider adding "null" to the method categories? 
Is "ligand-based" clear enough? Participants may assign methods that use ML-like ligand property based methods and shape-based methods. Where would be pharmacophore models or ROCS-like shape similarity methods categorised? 

I think our working definitions of these category names should be added to challenge instructions with examples if possible.

Reply:
Do you want to draft something along these lines? It would be helpful.